caluma/caluma_workflow/serializers.py

```diff
@@ -229,56 +171,24 @@ class CaseSerializer(SendEventSerializerMixin, serializers.ModelSerializer):
     )
 
     def validate(self, data):
-        form = data.get("form")
-        workflow = data["workflow"]
-
-        if form:
-            if (
-                not workflow.allow_all_forms
-                and not workflow.allow_forms.filter(pk=form.pk).exists()
-            ):
-                raise exceptions.ValidationError(
-                    f"Workflow {workflow.pk} does not allow to start case with form {form.pk}"
-                )
+        try:
+            data = models.Case.objects._validate(data)
```

Comment:
This is a private method call from "outside", which looks dangerous. See my other comment regarding validation